Makefile

```diff
@@ -47,7 +52,11 @@ GO_TAGS=$(if $(BUILDTAGS),-tags "$(BUILDTAGS)",)
 GO_LDFLAGS=-ldflags "-X $(PKG)/version.Version=$(VERSION) -X $(PKG)/version.Revision=$(REVISION) -X $(PKG)/version.Package=$(PKG) $(EXTRA_LDFLAGS)"
 
 # Flags passed to `go test`
-TESTFLAGS ?=-parallel 8 -race -v
+ifeq ("$(GOOS)", "solaris")
```

Comment:
According to go test ``go test: -race and -msan are only supported on linux/amd64, freebsd/amd64, darwin/amd64 and windows/amd64`` So this is broken on a lot more than solaris.

Would you be willing to change this to only do -race on those combinations? 😄